### PR TITLE
Add schema based on diagram

### PR DESCRIPTION
**Pull request by topeogunleye**
### Added Changes
- Added schema based on diagram

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  UNIQUE (id),
+  type VARCHAR NOT NULL,
+  name VARCHAR NOT NULL
+);
+
+CREATE TABLE medical_histories (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  admitted_at TIMESTAMP NOT NULL,
+  patient_id BIGINT REFERENCES patients(id),
+  treatment_id BIGINT REFERENCES treatments(id),
```
Comment (Kingstalux) [inline schema_based_on_diagram.sql]: - [ ] Great job so far 👍 There is no `treatment_id` in `medical_histories` as that particular association is being handled by the `medical_histories_treatment`. Kindly rectify this issue 👍

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+CREATE INDEX fk_patient_id_medical_histories ON medical_histories(patient_id);
+CREATE INDEX fk_treatment_id_medical_histories ON medical_histories(treatment_id);
+
+CREATE TABLE invoices (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  UNIQUE(id),
+  total_amount DECIMAL NOT NULL,
+  generated_at TIMESTAMP NOT NULL,
+  payed_at TIMESTAMP NOT NULL,
+  fk_medical_history_id BIGINT NOT NULL,
```
Comment (Kingstalux) [inline schema_based_on_diagram.sql]: - [ ] Kindly change the `fk_medical_history_id` to simply just `medical_history_id` so as to follow the design exactly 😄

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @topeogunleye and @Dagic-zewdu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
149637855-7722d04c-b45d-4185-925f-7de047106461


## Highlights 👏 
✅ Good commit messages 👍 
✅ Professional README file 💯 
✅ Correct Gitflow used 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (topeogunleye) [reply schema_based_on_diagram.sql]: @Kingstalux, Thanks for the kind review. I have made changes and would really appreciate it if you check my changes before I submit my project for review again.

Comment (Kingstalux) [reply schema_based_on_diagram.sql]: Hello @topeogunleye your good to go 👍

**Review by Lameck1 — APPROVED** [review]
Hi, @topeogunleye & @Dagic-zewdu,

Tasks for this project have been successfully implemented 💯.

### Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

All the best with the next project.

You are free to merge this branch :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, remember to tag me in your question so I can receive the notification.**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**